src/javascripts/views/homeView.js

```diff
@@ -77,7 +75,19 @@ class HomeView {
         this.summaryTabEl.innerHTML = summaryTemplate;
     }
 
+    
     //----- EVENT HANDLER -----//
+    addDelegateShowInfo = (handlerTransactionInfo) => {
```

Comment:
Could you define more specific that what handler use for?

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!